www/%team/index.html.spt

```diff
@@ -37,7 +37,7 @@ title = name = team.name
 <div class="col0">
     {% if team.is_approved in (None, False) %}
     {% if team.is_approved == None %}unreviewed{% endif %}
-    {% if team.is_approved == False %}rejected{% endif %} |
+    {% if team.is_approved == False %}closed{% endif %} |
```

Comment:
I'm not sure if it's used anywhere, but it would be cleaner to use the `status` property - https://github.com/gratipay/gratipay.com/blob/8e369153ba1d5f878fa2948653bc8e4a01a1d13d/gratipay/models/team.py#L83